### DISCUSSION TITLE
"Max tab titles" & "sidebars" buttons missing

### DISCUSSION THREAD

**Discussion by churnish** [oxdc/obsidian-vertical-tabs#76] [Q&A] [answered]
In the image below, note that the plugin pane has five buttons on the top: 
![image](https://github.com/user-attachments/assets/3800b480-c81a-4b41-bb88-5bb30a95cbf2)

But I don't see two of those buttons ("max tab titles" and "sidebars") in my instance of Vertical Tabs. Where'd they go? 🤔

<img width="700" alt="Screen Shot 2024-12-14 at 15 30 41" src="https://github.com/user-attachments/assets/483b12fe-476a-4f16-9d16-7dce5a202ae7" />

**Comment by oxdc** [comment, marked as answer]
Hi @greetclammy ! I just haven't updated the image yet. Both options are moved to the settings.

If you want to maximize the active tab _and_ hide inactive ones, please toggle on the option "show active tabs only".

If you want to hide sidebar tabs in Vertical Tabs, please turn on "Hide sidebar tabs" in the settings.

**Reply by churnish** [reply]
Got it, thanks! To be honest, it would be nice to have those options easily accessible, like in the image. That's why I'm asking in the first place--seems quite handy! Could it be possible to add them back as an option? 🙏

**Reply by oxdc** [reply]
Hi @greetclammy ! I bring these buttons back in v0.12.3. Just turn on this option in the settings:

<img width="1019" alt="SCR-20241217-kptl" src="https://github.com/user-attachments/assets/0d4ec5e3-186a-4efa-9ed5-b638f35b8b48" />

**Reply by churnish** [reply]
Awesome, thank you!
